docs/howtoguides/use_pro_lxd_guests.rst

```diff
@@ -0,0 +1,25 @@
+How to get Ubuntu Pro on LXD guests
+***********************************
+
+Pro client v35 allows LXD guests to get Ubuntu Pro during boot. For the integration to
+work properly you need to:
+
+1) Have LXD installed from the `6/stable` channel
```

Comment:
Only this channel forever, or from this channel onwards?

Reply:
onwards. will fix the versions instead of channels.

Reply:
done